lib/config.go

```diff
@@ -117,7 +117,7 @@ func init() {
 
 	viperCfg.SetConfigName("config")
 	viperCfg.SetConfigType("yaml")
-	viperCfg.AddConfigPath(".")
+	viperCfg.AddConfigPath("./config.yaml")
```

Comment:
viperCfg.AddConfigPath(".")

Reply:
So I added this because of this [line in docker-compose](https://github.com/openshieldai/openshield/blob/9c4370e8de5ded968bec34ffbfcfc72b19715bfc/demo/docker-compose.yml#L18) which we bind a directory called "config.yaml" and then look into it for the config. So if the intent is to have the config file picked up from current directory then we need to change the way we mount configurations.

I can make the changes, just was confused on the intent.

Reply:
Got my answer from @pigri , will revert the changes to read it from the current directory and fix the docker-compose and readme to reflect the structure and file.